.dockerignore

```diff
@@ -1,2 +1,3 @@
 /.idea
+/open_api.key
```

Comment:
I think it should be `openai_api.key`. And btw for Advaisor I've choosed to make it global for the user:
```
mise@gol.in:~ $ ls -al ~/.advaisor/
celkem 16
drwx------  2 mise Domain Users 4096 úno 21 14:35 .
drwxr-xr-x 25 mise Domain Users 4096 kvě  7 18:54 ..
-rw-------  1 mise Domain Users   21 úno 20 16:56 gitlab_access_token
-rw-------  1 mise Domain Users   52 pro  5 17:04 openai_api_key
mise@gol.in:~ $
```

Reply:
Filename fixed. For this I choose store key local - one key for one purpose.